examples/xa-datasource/README.adoc

```diff
@@ -0,0 +1,331 @@
+Title: Injection Of Entitymanager
```

Comment:
add the following header:

:index-group: DataSources
:jbake-type: page
:jbake-status: status=published
= Injection Of Entitymanager

Reply:
Is corrected

Reply:
sorry , please remove tihis -> Title: Injection Of Entitymanager